source/manual/alerts/data-sync.html.md

```diff
@@ -4,9 +4,39 @@ title: Data sync
 section: Icinga alerts
 layout: manual_layout
 parent: "/manual.html"
-last_reviewed_on: 2020-02-24
+last_reviewed_on: 2020-03-05
 review_in: 6 months
 ---
+
+## govuk_env_sync (the new way)
+
+If you get an Icinga alert about a failing task, check `/var/log/syslog` and `/var/log/syslog.1` on the machine which runs the job. If the logs don't help, you can try re-running the sync job.
+
+The data sync operations are executed as cron-jobs attached to the `govuk-backup` user. Run the following commands to get an overview of the jobs being run on a machine.
+
+```bash
+$ sudo su - govuk-backup
```

Comment:
Optional improvement: can make this more auditable by not using a root shell, for example:

```suggestion
$ sudo crontab -lu govuk-backup
```

Reply:
Does the user need to execute the command as `govuk-backup`?